### PR TITLE
Clarify sampleUffMaskRCNN instructions

### PR DESCRIPTION
**Pull request by rmccorm4**
Users have been struggling a lot with the UffMaskRCNN sample. Hoping some of these tweaks will make it a little easier to run through.

1. Clarify instructions with an explicit `$DATA_DIR` for easier copy/paste
2. Replace misleading build instructions for individual sample, they don't seem correct standalone
    * https://github.com/NVIDIA/TensorRT/issues/123#issuecomment-551428910
3. Change `tensorflow < 2.0` to `tensorflow < 1.15` to avoid `addV2` issues seen here: https://github.com/NVIDIA/TensorRT/issues/125#issuecomment-533757881. Related issues:
    * https://github.com/NVIDIA/TensorRT/issues/123
    * https://github.com/NVIDIA/TensorRT/issues/125
    * https://github.com/NVIDIA/TensorRT/issues/132
4. add TF1.15 addV2 known issue in README

Signed-off-by: Ryan McCormick <rmccormick@nvidia.com>

### CODE REVIEW

**Comment by Apsu95** [comment]
Hello all,

I successfully converted my model to UFF.

When I run the sample, I have this error:

&&&& RUNNING TensorRT.sample_maskrcnn # ./sample_maskRCNN -d /mnt/DTAA_data/DTAA/code/z662728/optimization/data/
[05/04/2020-09:24:19] [I] Building and running a GPU inference engine for Mask RCNN
[05/04/2020-09:24:21] [E] [TRT] UffParser: Validator error: mrcnn_detection_bboxes: Unsupported operation _SpecialSlice_TRT
&&&& FAILED TensorRT.sample_maskrcnn # ./sample_maskRCNN -d /mnt/DTAA_data/DTAA/code/z662728/optimization/data/

How did you solve it? @rmccorm4 @ShawnNew I would be very grateful if you could please help me :)

Config:
TF-gpu 1.14.0
CUDA 10.0
TensorRT: 6.0.1.5
Keras 2.1.3

Thanks!

**Comment by kevinch-nv** [comment]
@rmccorm4 the instructions were updated in https://github.com/NVIDIA/TensorRT/pull/712. It looks like your comments about improving the clarity of the data directory is still applicable, can you rebase with ToT mastet?
